src/icons/fileIcons.ts

```diff
@@ -2325,5 +2325,13 @@ export const fileIcons: FileIcons = {
     },
     { name: 'typst', fileExtensions: ['typ'] },
     { name: 'ifanr-cloud', fileNames: ['.mincloudrc'] },
+    {
+      name: 'wechat-miniprogram',
+      fileNames: [
```

Comment:
Unfortunately these file names are not only used for wechat projects. There might be multiple projects outside, having a "project.config.json" file but are not related to wechat. This is why I don't want to have icon associations for files where theres's a high likelilhood, that other projects use these files as well.